### PR TITLE
Version Update from v0.8.0 to v0.9.0

### PR DESCRIPTION
**Pull request by leekahung**
# PR Details

Version update here will be from v0.8.0 to v0.9.0 for the Master branch. Will create a new v0.9.0 Release after we got this merged into Master.

Major updates since last version includes:
- Adding logo to navbar/ favicon #348
- Rewrite clients with react query #362
- Revamp status notifications #331
- Private profile info and fixing profile bug #379
- Refactor container creation #384
- Create User Registration Page #378
- Delete old notification system #392
- Putting SetAclPermissions into Modal #388
- Create message read status (#400)
- Message preview update #390

Bug fixes:
- Profile fails to load client info correctly due to incorrect parameters #387
- Bug fix for contacts page #394
- Profile bug for non-existent webIds #402
- PASS logo production bug #407

Other fixes:
- Minor clean-up to root files #380
- Refactor styling from stylecss into themejs #383
- Only parse valid contacts #395
- Clean-up Profile Components #401
- Clean up Development v9 #409
- Update to PR template for GitHub #424

### CODE REVIEW

**Comment by xscottxbrownx** [comment]
I'd prefer we merge in Andy's latest work on message preview too.    Brandon's work on sharing modals too.

If we could get both of those in, the app will start looking pretty clean for an MVP.

File: src/components/Contacts/ContactListTable.jsx [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
+ * ContactListTable Component - Component that generates table of contacts from data within ContactList
  *
- * @memberof Clients
- * @name ClientListTable
- * @param {clientListTableProps} Props - Props for ClientListTableRow
- * @returns {React.JSX.Element} The ClientListTableRow Component
+ * @memberof Contacts
+ * @name ContactListTable
+ * @param {contactListTableProps} Props - Props for ContactListTableRow
+ * @returns {React.JSX.Element} The ContactListTableRow Component
```
Comment (xscottxbrownx) [inline src/components/Contacts/ContactListTable.jsx]: not accurate.  It returns a table.
not sure if line 32 is accurate either.

**Comment by leekahung** [comment]
> I'd prefer we merge in Andy's latest work on message preview too.    Brandon's work on sharing modals too.
> 
> If we could get both of those in, the app will start looking pretty clean for an MVP.

Yeah, that's the plan. Keeping this a draft for the time being.

File: src/components/NavBar/NavbarDesktop.jsx [hunk 2]
```diff
@@ -13,8 +13,9 @@ import Menu from '@mui/material/Menu';
 import MenuItem from '@mui/material/MenuItem';
 import NotificationsIcon from '@mui/icons-material/Notifications';
 import Toolbar from '@mui/material/Toolbar';
-import Typography from '@mui/material/Typography';
 import { useTheme } from '@mui/material/styles';
+import Link from '@mui/material/Link';
```
Comment (xscottxbrownx) [inline src/components/NavBar/NavbarDesktop.jsx]: Pretty positive this should be the `Link` imported from `react-router-dom` so there is no network request made. Let react-router interrupt and send user where we want.

File: src/components/NavBar/NavbarLoggedOut.jsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import AppBar from '@mui/material/AppBar';
 import Box from '@mui/material/Box';
 import Toolbar from '@mui/material/Toolbar';
-import Typography from '@mui/material/Typography';
+import Link from '@mui/material/Link';
```
Comment (xscottxbrownx) [inline src/components/NavBar/NavbarLoggedOut.jsx]: Pretty positive this should be the `Link` imported from `react-router-dom` so there is no network request made. Let react-router interrupt and send user where we want.

File: src/components/NavBar/NavbarMobile.jsx [hunk 4]
```diff
@@ -57,9 +56,12 @@ const NavbarMobile = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-            PASS
-          </Typography>
+          <img
```
Comment (xscottxbrownx) [inline src/components/NavBar/NavbarMobile.jsx]: This should also have a `react-router` `Link` around it and so when user clicks it takes back to home - just like the other `Navbar`s.

**Comment by leekahung** [comment]
Pulled in a few of the other minor PRs into this update. Opening this up for review.

File: src/components/NavBar/NavbarDesktop.jsx [hunk 5]
```diff
@@ -61,9 +62,14 @@ const NavbarDesktop = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-            PASS
-          </Typography>
+          <Link href="/">
+            <img
+              src="src/assets/pass-logo.png"
```
Comment (leekahung) [inline src/components/NavBar/NavbarDesktop.jsx]: Static image for logo not loading/rendering in production. Likely it's not being sourced correctly.

Comment (leekahung) [reply src/components/NavBar/NavbarDesktop.jsx]: Created issue (#406) and PR (#407) to get this fixed.

File: src/components/Messages/MessagePreview.jsx [hunk 6]
```diff
@@ -25,46 +34,86 @@ import { NewMessageModal } from '../Modals';
...
-            </Box>
-          </Box>
+
+              {showContents && (
+                <Grid item xs={12}>
+                  <Divider />
+                  {message.message.split('\n').map((line, index) => (
+                    <Typography
+                      sx={{
+                        padding: '10px 5px 10px 5px'
```
Comment (xscottxbrownx) [inline src/components/Messages/MessagePreview.jsx]: could be just `padding: '10px 5px'` but not a big deal

File: src/theme.js [hunk 7]
```diff
@@ -29,6 +29,29 @@ const theme = createTheme({
...
+  // color properties TBD for status snackbar severieties
+  components: {
+    MuiAlert: {
+      styleOverrides: {
+        filledSuccess: {
+          backgroundColor: 'green',
+          color: 'white'
+        },
+        filledError: {
+          backgroundColor: '#red',
```
Comment (xscottxbrownx) [inline src/theme.js]: `#`?

File: src/components/Contacts/ContactListTable.jsx (hunk 1 above)
Comment (leekahung) [reply src/components/Contacts/ContactListTable.jsx]: Fixed in PR #409

File: src/components/NavBar/NavbarDesktop.jsx (hunk 2 above)
Comment (leekahung) [reply src/components/NavBar/NavbarDesktop.jsx]: Made the change to utilize React Router Link instead (see corrections in PR #409)

File: src/components/NavBar/NavbarLoggedOut.jsx (hunk 3 above)
Comment (leekahung) [reply src/components/NavBar/NavbarLoggedOut.jsx]: See comment above about Link

File: src/components/NavBar/NavbarMobile.jsx (hunk 4 above)
Comment (leekahung) [reply src/components/NavBar/NavbarMobile.jsx]: Have fixed this to include the React Router Link

File: src/components/Messages/MessagePreview.jsx (hunk 6 above)
Comment (leekahung) [reply src/components/Messages/MessagePreview.jsx]: Updated the padding to `padding: '10px 5px'` in PR #409

File: src/theme.js (hunk 7 above)
Comment (leekahung) [reply src/theme.js]: Removed the `#` from the color (see PR #409)

**Comment by leekahung** [comment]
For this version update, PR #411 will be the last update to keep this from growing any further. Think we'll have existing PRs in the current pipeline moved to the next update.

**Review by xscottxbrownx — APPROVED** [review]

**Comment by andycwilliams** [comment]
Don't forget to include #390 in the list of updates.

Also, not sure whether it would be done in version update PRs but I just wanted to mention that the list of contributors in the readme is out of date. We have quite a few new people who ought to be included. I'm not sure whether that would necessitate its own PR.

**Review by andycwilliams — APPROVED** [review]

**Comment by leekahung** [comment]
> Don't forget to include #390 in the list of updates.

I'll have it updated in the PR description.

> Also, not sure whether it would be done in version update PRs but I just wanted to mention that the list of contributors in the readme is out of date. We have quite a few new people who ought to be included. I'm not sure whether that would necessitate its own PR.

Think we can update this in the next update with a PR.

**Comment by leekahung** [comment]
Hey @timbot1789 , @xscottxbrownx and I had recently found one of the merge checks failing in the PR to Master. The strange part is that we can't seem to locate the offending file in either Development or Master branches. Is there a way to handle this or we could just disregard it?

**Comment by leekahung** [comment]
Alright, gonna try and merge PR #424 (which is mostly administrative changes) into branch to see if the CI check gets fixed.

**Comment by leekahung** [comment]
> Hey @timbot1789 , @xscottxbrownx and I had recently found one of the merge checks failing in the PR to Master. The strange part is that we can't seem to locate the offending file in either Development or Master branches. Is there a way to handle this or we could just disregard it?

Never mind. Think we got this fixed!

**Comment by leekahung** [comment]
Alright, think we could get this version update made now. Anything else we need to fix/amend we could do it in subsequent Issues/PRs.

**Pull request merged**